marathon/client.py

```diff
@@ -167,16 +178,29 @@ def list_apps(self, cmd=None, embed_tasks=False,
             apps = [o for o in apps if getattr(o, k) == v]
         return apps
 
-    def get_app(self, app_id, embed_tasks=False):
+    def get_app(self, app_id, embed_tasks=False, embed_counts=False,
+                embed_deployments=False, embed_readiness=False,
+                embed_last_task_failure=False, embed_failures=False,
+                embed_task_stats=False):
         """Get a single app.
 
         :param str app_id: application ID
         :param bool embed_tasks: embed tasks in result
+        :param bool embed_counts: embed all task counts
+        :param bool embed_deployments: embed all deployment identifier
+        :param bool embed_readiness: embed all readiness check results
+        :param bool embed_last_task_failure: embeds the last task failure
+        :param bool embed_failures: shorthand for embed_last_task_failure
+        :param bool embed_task_stats: embed task stats in result
 
         :returns: application
         :rtype: :class:`marathon.models.app.MarathonApp`
         """
-        params = {'embed': 'apps.tasks'} if embed_tasks else {}
+        params = {}
+        embed_param = self._parse_embed_params()
```

Comment:
I know it's significantly more verbose, but I'd much prefer you do the following to using inspect:
```
embed_params = {
      'app.tasks': embed_tasks,
      'app.counts': embed_counts,
      'app.deployments': embed_deployments
      ....
}
filtered_embed_params = { k: v for k,v in embed_params if v }
params['embed'] = filtered_embed_params
```

Personally I think it makes it more explicit what is 'really' happening here, rather than hiding that complexity in the ``_parse_embed_params`` function (and it introspecting the current frame).